config_distro.py

```diff
@@ -0,0 +1,3 @@
+import os
+
+DATA_DIR = os.environ["XDG_CONFIG_HOME"]
```

Comment:
XDG_CONFIG_HOME isn't typically defined as an env var. This needs a fallback to $HOME/.config if the env var is non existent

Reply:
Also os.environ will raise a keyerror then, which needs to be handled.

Reply:
Ah nevermind I realised this will be inside Flatpak where it is always set.